library/src/main/kotlin/it/krzeminski/githubactions/domain/RunnerType.kt

```diff
@@ -4,22 +4,25 @@ package it.krzeminski.githubactions.domain
  * Types of GitHub-hosted runners available. Should be kept in sync with the official list at
  * https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners
  */
-enum class RunnerType {
+sealed class RunnerType {
```

Comment:
nit: could be an interface - will remove the noise of writing `()` at the end of `RunnerType` for each type :smile: 

```suggestion
sealed interface RunnerType {
```

Reply:
Nice